test/Directory.Build.props

```diff
@@ -2,10 +2,9 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <DefaultTestTargetFramework>netcoreapp2.1</DefaultTestTargetFramework>
-    <DefaultTestTargetFramework Condition="$(Configuration.EndsWith('-netfx'))">net462</DefaultTestTargetFramework>
-    <DefaultTestTargetFramework Condition="'$(UseIntrinsics)' == 'true'">netcoreapp3.0</DefaultTestTargetFramework>
-    <TargetFramework>$(DefaultTestTargetFramework)</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework Condition="$(Configuration.EndsWith('-netfx'))">net462</TargetFramework>
```

Comment:
(not part of this change) - do you remember why we choose `net462` instead of `net461`?

Reply:
I remember trying it with net471 and santi suggested to try with net462 to just make the build successfull. It was a while back. and i didnt try building with any other versions

Reply:
If net461 works we should probably use that everywhere, since that supports NET Standard 2.0 it seems like the natural choice.